sep-template.md

```diff
@@ -1,35 +1,42 @@
-Copied from [eip-X](https://github.com/ethereum/EIPs/blob/master/eip-X.md).
-
 ## Preamble
 
 ```
-SEP: <to be assigned>
-Title: <SEP title>
-Author: <list of authors' names and optionally, email addresses>
+SEP: To Be Assigned
+Title: <SEP title - keep it short and sweet>
+Author: <list of authors' names and optionally, email addresses, separated by commas>
+Track: <Informational or Standard>
 Status: Draft
 Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+Discussion: <link to where discussion for this SEP is taking place, typically the mailing list>
 ```
 
 ## Simple Summary
-"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the SEP.
+"If you can't explain it simply, you don't understand it well enough." Provide a simplified and
+layman-accessible explanation of the SEP.
+
+## Motivation
+Should clearly explain why the existing protocol specification is inadequate to address the problem
+that the SEP solves. SEP submissions without sufficient motivation may be rejected outright.
 
 ## Abstract
 A short (~200 word) description of the technical issue being addressed.
 
-## Motivation
-Should clearly explain why the existing protocol specification is inadequate to address the problem that the SEP solves. SEP submissions without sufficient motivation may be rejected outright.
-
 ## Specification
 The technical specification should describe the syntax and semantics of any new feature.
 
 ## Rationale
```

Comment:
Having both Motivation and Rational is confusing for me. Maybe make it more clear how they're different?

Reply:
I changed Rational to `Design Rationale` as Motivation is supposed to be "What's the problem that needs to be addressed" where as Design Rationale is supposed to be "Why this design over others to solve the problem". Let me know if you still think it's confusing, open to suggestions.